examples/agones-xonotic-xds/xds-control-plane.yaml

```diff
@@ -103,7 +103,7 @@ spec:
       annotations:
         prometheus.io/scrape: "true"
         prometheus.io/path: /metrics
-        prometheus.io/port: "9091"
```

Comment:
I have been wondering if we should switch the admin port back to this since it is the default, but then again you have to specify it anyway,

Reply:
Hunting around a bit, I _think_ prometheus will default to `80` (it's not the best documented) -- and looking around at a few different services, it seems like the port changes to all kind of things.

So no strong opinions on what the port ends up being.